src/components/Tables/Table.tsx

```diff
@@ -164,7 +175,8 @@ const Table: React.FC<Props> = ({
     };
 
     const toggleAllCheckBox = (): void => {
-        setSelectCheckBoxes(new Array<boolean>(data.length).fill(!selectAllCheckBox));
+        const newSelection = new Array<boolean>(data.length).fill(!selectAllCheckBox);
+        setSelectCheckBoxes(newSelection);
```

Comment:
Is this just a new function? Or is it the same as the one with the underscore at the front?